packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -406,9 +459,12 @@ class HorizontalOverflow extends Foundation<
     };
 
     /**
-     * Handler for the click event fired after next or previous has been clicked
+     * Returns the items width and total width of the scroll region
      */
-    private handleClick(direction: ButtonDirection): void {
+    private getItemsWidthAndTotalWidth(): {
```

Comment:
doesn't seem to specify a return type

Reply:
It does? 
```
private getItemsWidthAndTotalWidth(): {
        availableWidth: number;
        itemWidths: number[];
    } {...}

Reply:
I went ahead and made this an interface to solve the below issue and clear this up.

Reply:
Ah - I think this is prettier showing up here because an object is being specified as the return object and it looks like the function...

_Should we create an interface here?_ Thoughts?